File: src/main/java/refactown/cleancode/c02names/BadCode.java
package refactown.cleancode.c02names;

import java.lang.reflect.Field;
import java.time.LocalDate;
import java.time.Month;
import java.util.ArrayList;
import java.util.List;

public class BadCode {
    // nomes genéricos
    int d = 5;
    List list = new ArrayList<Field>();
    void func(int n){
        for (int i=0; i < 10; i++){
            if (n == 3) System.out.println("count:" + i + " = 3");
        }
    }

    // Nomes não pronunciáveis
    LocalDate genymdhms=  LocalDate.now();
    LocalDate modymdhms=  LocalDate.now();
    int pszqint = 102;

    // Número mágico - difpicil de encontrar depois
    public static boolean podeSerEireli(int socios){
        return socios <= 3;
    }

    // Notação húngara
    String strNome = "Fulano";
    int intIdade = 30;
    LocalDate dtNascimento = LocalDate.of(1990, Month.APRIL, 12);

    // Etc
    private IPessoa pessoa  =new Pessoa();
    EnderecoDoCliente enderecoDeCobrancaDoCliente=new EnderecoDoCliente();
    EnderecoDoCliente enderecoDeEntregaDoCliente= new EnderecoDoCliente();

}

File: src/main/kotlin/refactown/cleancode/c02names/BadCode.kt
package refactown.cleancode.c02names

import java.lang.reflect.Field
import java.time.LocalDate
import java.time.Month
import java.util.*

// Nomes genéricos
val d: Int = 5 // tempo decorrido em dias
val list = ArrayList<Field>() // field list
fun func(n: Int){
    for (i in 1..10)
        if (n == 3)(println("count:$i = 3"))
}

// Nomes não pronunciáveis
val genymdhms=  LocalDate.now();
val modymdhms=  LocalDate.now();
val pszqint = 102;

// Número Mágico - difícil de encontrar depois
fun podeSerEireli(socios: Int) = socios <= 3

// Notação Húngara
val strNome = "Fulano"
val intIdade = 30
val dtNascimento = LocalDate.of(1990,Month.APRIL, 12)

// Etc
private val pessoa : IPessoa = Pessoa()
val enderecoDeCobrancaDoCliente= EnderecoDoCliente()
val enderecoDeEntregaDoCliente= EnderecoDoCliente()







class EnderecoDoCliente
internal interface IPessoa
open class Pessoa: IPessoa


